<?xml version="1.0" encoding="UTF-8"?>
<!--
Copyright (C) 2019  Christian Schärf

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>MeasuredValuesForm</class>
 <widget class="QWidget" name="MeasuredValuesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>280</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="4">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="2">
    <widget class="QPushButton" name="loadButton">
     <property name="text">
      <string>Aus Datei laden...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>In Datei speichern...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="removeButton">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Löschen</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" rowspan="4">
    <widget class="QTableView" name="list">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
